### PR TITLE
Sct 76 modal validation

### PR DESCRIPTION
**Pull request by JaycobDuffel**

### CODE REVIEW

**Review by DawsonDease — APPROVED** [review]

File: scutl.py [hunk 1]
```diff
@@ -723,10 +730,10 @@ def handle_modal(body, ack, client, logger):
 def recurring_modal_submission(ack, body, client, logger):
     ack()
     if date_time_days['date'] is None:
-        date_time_days['date'] = body['view']['blocks'][0]['accessory']['initial_date']
+        date_time_days['date'] = body['view']['blocks'][0]['element']['initial_date']
 
     requests.post(
-        "{}/events/".format(os.environ.get("API_URL")),
+        "{}/organiztations/1/events/".format(os.environ.get("API_URL")),
```
Comment (katiemcgoff) [inline scutl.py:736]: typo in organizations

Comment (DawsonDease) [reply scutl.py:736]: good catch

Comment (katiemcgoff) [reply scutl.py:736]: Also - organization should be removed from data in this call as well.

**Review by katiemcgoff — CHANGES_REQUESTED** [review]
Manually tested, looks great.

**Review by katiemcgoff — APPROVED** [review]

**Pull request merged**